draft-ietf-acme-acme.md

```diff
@@ -982,65 +982,73 @@ A client may wish to change the public key that is associated with a
 registration in order to recover from a key compromise or proactively mitigate
 the impact of an unnoticed key compromise.
 
-To change the key associated with an account, the client sends a POST request
-containing a key-change object with the following fields:
+To change the key associate with an account, the client first constructs a
+key-change object describing the change that it would like the server to make:
 
 oldKey (required, JWK):
-: The JWK representation of the original key (i.e., the client's current account
+: The JWK thumbprint of the original key (i.e., the client's current account
 key)
 
 newKey (requrired, JWK):
-: The JWK representation of the new key
+: The JWK thumbprint of the new key
```

Comment:
Good point.  I think we probably should, as we do for key authorizations below (vs. specifying one in the object).  We can update later by defining a new `key-change` endpoint.  

We also need to specify that the thumbprint is base64url-encoded, since that's not in RFC 7638 either.  Maybe the thing to do is just to say somewhere at the beginning that when we say "thumbprint" we mean "... with SHA-256, base64url-encoded".